File: app/src/main/java/net/osdn/gokigen/blecontrol/lib/ble/connection/CameraBleSetArrayItem.java
package net.osdn.gokigen.blecontrol.lib.ble.connection;

class CameraBleSetArrayItem
{
    private final String dataId;
    private String btName;
    private String btPassCode;
    private String information;

    CameraBleSetArrayItem(String dataId, String name, String passCode, String information)
    {
        this.dataId = dataId;
        this.btName = name;
        this.btPassCode = passCode;
        this.information = information;
    }

    String getDataId()
    {
        return (dataId);
    }

    String getBtName()
    {
        return (btName);
    }

    String getBtPassCode()
    {
        return (btPassCode);
    }

    String getInformation()
    {
        return (information);
    }
}

File: app/src/main/java/net/osdn/gokigen/blecontrol/lib/ble/connect/CameraBleSetArrayItem.kt
package net.osdn.gokigen.blecontrol.lib.ble.connect

class CameraBleSetArrayItem
    (
    private val dataId: String?,
    private val btName: String?,
    private val btPassCode: String?,
    private val information: String?
) {
    fun getDataId(): String? {
        return (dataId)
    }

    fun getBtName(): String? {
        return (btName)
    }

    fun getBtPassCode(): String? {
        return (btPassCode)
    }

    fun getInformation(): String? {
        return (information)
    }
}
